### ISSUE TITLE
Error during npm install

### ISSUE THREAD

**Issue by lexon0011** [compodoc/gulp-compodoc#1]
I get the following error during the installation:
`npm ERR! path D:\node_modules\gulp-compodoc\node_modules\compodoc\bin\index-cli.js`
<img width="487" alt="gulp-compodoc" src="https://cloud.githubusercontent.com/assets/16290662/23798902/dc31bce6-05a6-11e7-8407-9bd1147d1969.PNG">

I use the latest version of NPM and compodoc

Could you tell me, how I solve this problem? Thanks!

**Comment by vogloblinsky** [comment]
As explain in the README, gulp-compodoc is in the scope @compodoc

```
npm install --save-dev @compodoc/gulp-compodoc
```

**Comment by lexon0011** [comment]
It doesn't works. 

For you information: on the [NPM](https://www.npmjs.com/package/gulp-compodoc) site is an other install command.

Here is the log file of the installation:
[2017-03-10T15_15_05_700Z-debug.zip](https://github.com/compodoc/gulp-compodoc/files/834287/2017-03-10T15_15_05_700Z-debug.zip)

**Comment by vogloblinsky** [comment]
Fixed with 0.0.3

```
npm install --save-dev @compodoc/gulp-compodoc
```

**Comment by lexon0011** [comment]
Thanks, works! 

One additional info: The gulp task works only with the installation of event-stream - It would be nice, if your package has a dependency to this package, so that it would be installed automatically during the gulp-compodoc installation. 

Thanks for your great work! Compodoc is really nice!

**Comment by vogloblinsky** [comment]
Fixed in 0.0.4
